public/components/trace_analytics/requests/request_handler.ts

```diff
@@ -34,7 +34,7 @@ export async function handleDslRequest(
     body = { ...bodyQuery, index: mode === 'jaeger' ? JAEGER_INDEX_NAME : DATA_PREPPER_INDEX_NAME };
   }
   if (timeout) {
-    const id = setTimeout(() => setShowTimeoutToast!(), 30000);
+    const id = setTimeout(() => setShowTimeoutToast!(), 25000); // 25 seconds
```

Comment:
setShowTimeoutToast is optional but asserted to be non null here?

Reply:
Thanks, took @Swiddis 's suggestion to only call it if the function itself is not null, instead of unnecessarily associating it with another variable

Reply:
Regarding your second point, I think we aligned to not stopping the request as to preserve as much functionality as possible, in case their OSD is configured to wait longer for requests. It is mainly informational to let them know about the resources being used by these requests